tests/tools/importexport/__init__.py

```diff
@@ -7,3 +7,19 @@
 # For further information on the license, see the LICENSE.txt file        #
 # For further information please visit http://www.aiida.net               #
 ###########################################################################
+"""Tests for AiiDA archive files (import, export)."""
+from aiida.backends.testbase import AiidaTestCase
+from aiida.tools.importexport import EXPORT_LOGGER, IMPORT_LOGGER
+
+
+class AiidaArchiveTestCase(AiidaTestCase):
+    """Testcase for tests of archive-related functionality (import, export)."""
+
+    @classmethod
+    def setUpClass(cls, *args, **kwargs):
+        """Only run to prepare an archive file"""
+        super().setUpClass()
+
+        # don't want output
+        EXPORT_LOGGER.setLevel('CRITICAL')
```

Comment:
Shouldn't you put it back to the correct level in the tear down class method? Otherwise other tests, later, using the same logger might not output relevant warnings